src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/ConditionMapper.java

```diff
@@ -172,9 +174,17 @@ static String getNextFieldPlaceholder(Map<String, String> expressionAttributeNam
         if (start == -1) {
             return Optional.empty();
         }
-        int end = conditionExpression.indexOf(" ", start + toFind.length()); // -1
-        return Optional.of(conditionExpression.substring(start + toFind.length(),
-            end == -1 ? conditionExpression.length() : end)); // ":value" // TODO add support for non-EQ operators
+
```

Comment:
re line 165: which comments?

Reply:
eek ... unintentionally removed comments, will fix

Reply:
fixed in ae504908b0fd0ce9ae5b2b5b580bdf0612ccfbcc and also added similar cueing that I had in other methods in this case